release_notes/ocp-4-8-release-notes.adoc

```diff
@@ -753,6 +760,11 @@ For more information, see xref:../scalability_and_performance/cnf-create-perform
 
 The xref:../scalability_and_performance/psap-node-feature-discovery-operator.adoc#node-feature-discovery-operator[Node Feature Discovery (NFD) Operator] is now available. Use it to expose node-level information by orchestrating Node Feature Discovery, a Kubernetes add-on for detecting hardware features and system configuration.
 
+[id="ocp-4-8-driver-toolkit"]
+==== The Driver Toolkit
+
+You can now use xref:../scalability_and_performance/psap-driver-toolkit.adoc#driver-toolkit[The Driver Toolkit] as a base image for driver containers so that you can enable special software and hardware devices on Kubernetes.
```

Comment:
```suggestion
You can now use xref:../scalability_and_performance/psap-driver-toolkit.adoc#driver-toolkit[the Driver Toolkit] as a base image for driver containers so that you can enable special software and hardware devices on Kubernetes.
```